### DISCUSSION TITLE
救命！我掉到异世界了。

### DISCUSSION THREAD

**Discussion by Ljh161** [share121/inter-knot#3167] [其他]
![landscape - su - flat](https://github.com/user-attachments/assets/a89d7cf7-3f43-4289-99c3-487a0d413cf7)
今天本来正常做委托，但是萝卜出问题了，再加上我的适应体质不高，就在空洞里乱跑，结果钻进了空间裂隙。我本以为我从空洞里跑出来了却发现这里根本不是丽都！这里的一切我都不认识，奇奇怪怪的生物到处都是还有各种机械构造和建筑，最重要的是这个鬼地方时不时的就会下雨，而且非常大，雨滴可以说是砸到身上。而且地上会往上漫水！还好我幸运的找到了一个没有雨的小型机械空腔，不然我就。。。
情况紧急，快救救我！

**Comment by x97-1** [comment]
### 叮！末日系统已绑定宿主。
